org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/RuntimeEnvironment.java

```diff
@@ -136,8 +136,8 @@ public URL getJavadocURL() {
 		}
 		URL url;
 		try {
-			url = new URL(javadoc);
-		} catch (MalformedURLException e) {
+			url = new URI(javadoc).toURL();
```

Comment:
toURL() is throwing java.lang.IllegalArgumentException: URI is not absolute, which isn't covered in the catch statement.

I think this is causing the test failure